spec/ttml1.xml

```diff
@@ -8094,6 +8094,30 @@ based timing or event based timing in <bibref ref="smil21"/>, where the specific
 <specref ref="time-expression-semantics-smpte"/> apply.</p>
 </item>
 </ulist>
+<p>The following example illustrates the timing semantics of anonymous spans that are children of sequential time containers. The words <emph>Hello</emph> and <emph>Allo</emph> are both contained in anonymous spans that are children of a sequential time container (the <code>p</code> element). As such, neither <emph>Hello</emph> nor <emph>Allo</emph> will be displayed since the implicit duration of the anonymous spans is 0. In contrast, the word <emph>Bonjour</emph> is contained within an anonymous span that is the child of a parallel time container (the <code>span</code> element). As such, <emph>Bonjour</emph> will be displayed since the implicit duration of the anonymous span is indefinite, resulting in the implicit duration of its <code>span</code>, <code>p</code>, <code>div</code> and <code>body</code> ancestors to also be indefinite. </p>
```

Comment:
I don't think it is true that the word "Bonjour" is contained in an anonymous span. The text in §7.1.5 p:

> If a sequence of children of a p element consists solely of character information items, then that sequence must be considered to be an anonymous span for the purpose of applying style properties that apply to span elements.

does not appear in 7.1.6 span so I don't think it applies.

Rather, the reason the word Bonjour appears indefinitely is because the explicit span's implicit duration is indefinite.

Reply:
Please disregard the previous comment - I've gone and checked and read it again properly and realise that 7.1.6 does have text that provides this behaviour:

> The span element accepts as its children zero or more elements in the Metadata.class element group, followed by zero or more elements in the Animation.class element group, followed by zero or more intermixed span elements, br elements, or **text nodes interpreted as anonymous spans**.

(my emphasis added)

Sorry for the noise.

Reply:
@nigelmegitt In fact, I think you were right in your first thought. In [9.3.3](https://www.w3.org/TR/ttml1/#semantics-region-layout-step-2) it actually says:

> for each span element whose child is a single anonymous span, replace the anonymous span with its sequence of child text nodes;

Reply:
@cconcolato This clause applies when constructing ISDs, which happens after timing is resolved.

Reply:
@palemieux I'm confused. The creation of 'anonymous spans' is only defined in [9.3.3](https://www.w3.org/TR/ttml1/#semantics-region-layout-step-2), therefore for me it does not make sense to talk about timing of an anonymous span, because as you say timing has been resolved already in 9.3.3.

Reply:
@palemieux TTML2 has a note that covers that:
> Note: A compliant presentation processor is permitted to perform the previous step (anonymous span generation) and this step (style resolution) as post-processing steps while performing the [construct intermediate document] procedure, in which case each intermediate synchronic document produced by 11.3.1.3 Intermediate Synchronic Document Construction will contain anonymously generated spans and computed 

but I couldn't find this text in TTML1. Maybe this should be backported.

Reply:
I think we need to understand why that note was added to TTML2.

In fact, TTML2 changed the sentence _the span element accepts as its children zero or more elements in the Metadata.class element group, followed by zero or more elements in the Animation.class element group, followed by zero or more intermixed span elements, br elements, or text nodes interpreted as anonymous spans_ to remove the mention of anonymous spans.

This is a problem since TTML2 also states that _If no timeContainer attribute is specified on a span element, then it must be interpreted as having parallel time containment semantics._, which leaves the implicit duration of a span undefined if the span element does not contain anonymous spans.

Reply:
It does seem that we should not have a discrepancy here between TTML1 and TTML2 if at all possible. I've added an agenda item for this, but perhaps it is something that can be covered in offline discussions before the F2F on Tuesday?